### PR TITLE
ECM-205: Added rest service to get enabled tasks. 

### PR DESCRIPTION
**Pull request by kraussj**
Service provides all tasks that are in the "enabled" state of the cmmn task lifecycle.

### CODE REVIEW

File: backend/src/main/java/org/educama/shipment/boundary/impl/ShipmentTaskBoundaryServiceImpl.java [hunk 1]
```diff
@@ -43,4 +44,25 @@
         }
         return shipmentTasks;
     }
```
Comment (larsduelfer) [inline backend/src/main/java/org/educama/shipment/boundary/impl/ShipmentTaskBoundaryServiceImpl.java]: Please remove the double blank lines and the other blank lines (if not really required).

**Comment by larsduelfer** [comment]
Hi, this branch needs rebasing as well.

**Comment by kraussj** [comment]
Removed the blank lines and rebased it.

**Comment by kraussj** [comment]
I recognized that the content of this pull request is very similar to the pull request ECM-187. Both implement a new service method in the backend to get all enabled tasks (execution items). 

The most important difference between both implementations is the usage of the DS object (ShipmentDS). We decided to use the already existing object for every tasks regardless of the type. Therefore the adaption to new types of tasks is easy and the object is universally applicable. In ECM-187 two new classes (enabled and historic) for data structures are introduced, which makes types of tasks distinguishable but increases the complexity of resource objects in backend and frontend. 

Before this pull request gets accepted it should be considered witch solutions is the best. If the ECM-187 gets picked this pull request is obsolete.

**Comment by tobiasschaefer** [comment]
As discussed today: I'm closing this PR now and we'll wait for you to create a new one.

**Pull request closed without merging**